homeassistant/components/picnic/services.yaml

```diff
@@ -0,0 +1,65 @@
+add_product:
+  name: Add product to the cart
+  description: >-
+    Adds a product to the cart based on a search string or product ID.
+    The search string is ignored if a product ID is given.
+    A product ID can be found by using the picnic.search service.
+
+  fields:
+    # Device/service selector
+    device_id:
+      name: Picnic service
+      description: The product will be added to the selected service, defaults to the first registered one.
+      required: false
+      selector:
+        device:
+          integration: picnic
+    # Product id text input
+    product_id:
+      name: Product ID
+      description: The product ID of a Picnic product.
+      required: false
+      example: "10510201"
+      selector:
+        text:
+    product_name:
+      # Product name text input
+      name: Product name
+      description: Search for a product and add the first result
+      required: false
+      example: "10510201"
+      # Input is a text string
+      selector:
+        text:
+    amount:
+      name: Amount
+      description: Amount to add of the selected product
+      required: true
+      default: 1
+      selector:
+        number:
+          min: 1
+          max: 50
+
+search:
+  # Service name/description
```

Comment:
A comment like this that is self explanatory can be removed as it is not adding value.